src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -89,12 +94,16 @@ def load_arguments(self, _):
         c.argument('enable_public_network', options_list=['--enable-public-network', '-e'], arg_type=get_three_state_flag(), is_preview=True,
                    help='When true, requests coming from public networks have permission to access this store while private endpoint is enabled. When false, only requests made through Private Links can reach this store.')
         c.argument('disable_local_auth', arg_type=get_three_state_flag(), is_preview=True, help='Disable all authentication methods other than AAD authentication.')
+        c.argument('retention_days', arg_type=retention_days_arg_type, is_preview=True)
+        c.argument('enable_purge_protection', arg_type=get_three_state_flag(), is_preview=True, help='Property specifying whether protection against purge is enabled for this App Configuration. Setting this property to true activates protection against purge for this App Configuration and its contents. Enabling this functionality is irreversible.')
```

Comment:
The lengths of all options ['--enable-purge-protection'] are longer than threshold 22. Argument enable_purge_protection must have a short abbreviation.

Reply:
Added a short abbreviation!